src/main/java/bareum/DeadlineTask.java

```diff
@@ -0,0 +1,61 @@
+package bareum;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class DeadlineTask extends Task {
+    LocalDate dueDate;
+
+    private DeadlineTask(boolean isDone, String description, LocalDate dueDate) {
+        super(isDone, description);
+        this.dueDate = dueDate;
+        this.isDone = false;
+        // add exceptions for time
```

Comment:
I think this is for a future addition right or is this misplaced

Reply:
Yep, I planned to add this in a future commit and left this as a reminder to myself :") Thanks for the feedback!